packages/core/admin/server/middlewares/rateLimit.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const utils = require('@strapi/utils');
+const { toLower } = require('lodash/fp');
+
+const { RateLimitError } = utils.errors;
+
+module.exports =
+  (config, { strapi }) =>
+  async (ctx, next) => {
+    const ratelimit = require('koa2-ratelimit').RateLimit;
+
+    const userEmail = toLower(ctx.request.body.email) || 'unknownEmail';
+
+    return ratelimit.middleware({
+      interval: { min: 5 },
+      max: 5,
+      prefixKey: `${userEmail}:${ctx.request.path}:${ctx.request.ip}`,
+      handler() {
+        throw new RateLimitError();
+      },
+      ...strapi.config.get('admin.ratelimit'),
```

Comment:
What would be the recommended way of disabling this entirely (for someone who didn't want to ratelimit)? Setting interval: 0? Or might we want to let someone set `admin.ratelimit: false` in their config and check that at the beginning here?

I'm specifically thinking of documentation here, and how to ensure someone who has disabled it doesn't accidentally get it turned back on if we change this implementation.

Reply:
Good question, I could add a simple enabled t/f here.

Reply:
Do you want to add it now or later ? To know if we can merge the PR